serverless.yaml

```diff
@@ -215,6 +219,17 @@ functions:
     role: UpdateStatusLambdaRole
     handler: bulkExport/index.updateStatusStatusHandler
 
+  uploadGlueScripts:
+    timeout: 30
+    memorySize: 192
+    runtime: nodejs12.x
+    description: 'Upload glue scripts to s3'
+    role: UploadGlueScriptsLambdaRole
+    handler: bulkExport/uploadGlueScriptsToS3.handler
+    disableLogs: true # needed to avoid race condition error "Resource of type 'AWS::Logs::LogGroup' already exists" since the custom resource lambda invocation may create the log group before CFN does

```

Comment:
Interesting - does this mean we will not have logs for the UploadGlueScriptsLambda? If we still do console.log() in the lambda does that just write locally and we are unable to see it?

Reply:
No, it just means that Serverless won't explicitly create the LogGroup by generating an `AWS::Logs::LogGroup` resource. 

Lambda functions create the corresponding LogGroup/LogStream when they are invoked for the first time if it does not exist already.

We'll have Lambda logs as usual.